maas/client/viscera/tests/test_machines.py

```diff
@@ -92,3 +92,9 @@ def test__allocate(self):
             tags=['foo', 'bar'],
             not_tags=['baz'],
         )
+
+    def test__power_parameters(self):
+        Machines = make_origin().Machines
+        Machines._handler.power_parameters.return_value = {}
```

Comment:
The return value from MAAS is, I think, a dict of dicts, so this ought to do something similar. In addition, it ought to show that the value returned from "the server" is given to the caller.

```python
power_parameters = {make_name("system_id"): {"key": make_name("value")}}
Machines._handler.power_parameters.return_value = power_parameters
self.assertThat(
    Machines.get_power_parameters(),
    Equals(power_parameters))
Machines._handler.power_parameters.assert_called_once_with(id=[])
```

A second test should show that `system_ids` is passed through.

```python
power_parameters = {
    make_name("system_id"): {"key": make_name("value")}
    for _ in range(3)
}
Machines._handler.power_parameters.return_value = power_parameters
self.assertThat(
    Machines.get_power_parameters(system_ids=power_parameters.keys()),
    Equals(power_parameters))
Machines._handler.power_parameters.assert_called_once_with(
    id=list(power_parameters))
```